MetaMask/metamask-mobile
# test: complete quiz to reveal SRP

## **Description**

This PR updates the SRP quiz to move it from smoke to regression, includes revealing the SRP text and copying to clipboard.  We also make sure that the SRP is not revealed with an incorrect password.

~~Also corrects an earlier spelling mistake in SecurityQuizModal.js~~
Updated selector names used in the earlier SRP quiz test.


## **Related issues**

Fixes: [561](https://github.com/MetaMask/accounts-planning/issues/561)

## **Manual testing steps**

On local machine run on this branch...
Android: `yarn test:e2e:android:debug:build` then `yarn test:e2e:android:debug:run e2e/specs/accounts/reveal-secret-recovery-phrase.spec.js`
iOS: `yarn test:e2e:ios:debug:build` then `yarn test:e2e:ios:debug:run e2e/specs/accounts/reveal-secret-recovery-phrase.spec.js`


pr_regression_e2e_pipeline: [*PASSED](https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/feca871d-263c-4962-91f1-5aebaeec5a57) *Note that regression test that is failing (Regression Multiple Swaps from Actions should Swap 1 'ETH' to 'WETH') also failed on 8/26  & 8/27 nightlys and it not related to my changes here.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
